tests/phpunit/elementor/includes/test-utils.php

```diff
@@ -83,6 +83,22 @@ public function test_should_throw_error_because_urls_are_invalid() {
 		Utils::replace_urls( 'elementor', '/elementor' );
 	}
 
+	public function test_replace_urls() {
+		$this->act_as_admin();
+		global $wpdb;
+
+		//Arrange
+		$old_home_url = 'http://example.org';
+		$new_home_url = 'http://example2.org';
+		$expected_query = "UPDATE wptests_postmeta SET `meta_value` = REPLACE(`meta_value`, 'http:\\\/\\\/example.org', 'http:\\\/\\\/example2.org') WHERE `meta_key` = '_elementor_data' AND `meta_value` LIKE '[%';";
+		$get_query = true;
+
+		//Act
+		$last_query = Utils::replace_urls( $old_home_url, $new_home_url, $get_query );
```

Comment:
```suggestion
		Utils::replace_urls( $old_home_url, $new_home_url, $get_query );
		
		$last_query =  $wpdb->last_query;
```

use that instead of changing the method

Reply:
This is not working this is not the last query. 
The function is triggering more queries. so i lost the actual query that i need. this is why i add this code in this specific location.